File: programme/validate-palindrome/validate-palindrome.py
# Write a programme to Validate Palindrom string

def validPalindrome(s):
    lowString = s.lower()

    reverse_string = lowString[::-1]

    return s == reverse_string


example1 = "level"
print("Input  :", example1)
print("Output :", validPalindrome(example1))

example2 = "coding"
print("\nInput  :", example2)
print("Output :", validPalindrome(example2))

File: programme/validate-palindrome/validate-palindrome.cs
// Write a programme to validate palindrome
using System;

namespace validPalindrome
{
    internal class Program
    {
        static bool validPalindrome(string s)
        {
            string lowString = s.ToLower();

            string reverseString = string.Empty;
            for (int i = lowString.Length - 1; i >= 0; i--)
            {
                reverseString += lowString[i];
            }
            return s == reverseString;
        }

        static void Main(string[] args)
        {
            string example1 = "level";
            Console.WriteLine($"Input  : {example1}");
            Console.WriteLine($"Output : {validPalindrome(example1)}");
        }
    }
}
